packages/cli/src/util/createServer.ts

```diff
@@ -78,13 +79,14 @@ async function createPrismServerWithLogger(options: CreateBaseServerOptions, log
 
   operations.forEach(resource => {
     const path = pipe(
-      createExamplePath(resource),
+      createExamplePath(resource, attachTagsToParamsValues),
```

Comment:
Why not push only one function to `createPathExample` which will cover the whole colorizing process? WDYT?

Reply:
@StefanDywersant, do you mean `createExamplePath` and `transformPathParamsValues`? If yes, then I don't how we could do this with only one function. It would be possible if https://github.com/stoplightio/prism/blob/d13c4742c4b21d1f5da0c8f013f2a3e091ec353d/packages/cli/src/util/paths.ts#L36 was not serializing the path. Then, the colorization could take place inside `attachTagsToParamsValues` (under a different name of course ;>). But maybe you had something else in mind? Or maybe you have some solution in mind?

Reply:
@XVincentX, I don't think this is weird. With this solution, `createExamplePath`, is open for extension because the dependency is injected and hence it is loosely coupled. But if you really rather see `createExamplePath` strongly dependent on `attachTagsToParamsValues`, then LMK.

Reply:
I understand your point — but on the other hand at the moment I really do not see any of us extending this function and at this point it's just an additional complexity when reading the code. We can put it back in case the needs arises with the pass of the time.